examples/.cargo/config.toml

```diff
@@ -33,7 +33,7 @@ PASSWORD = "PASSWORD"
 STATIC_IP = "1.1.1.1 "
 GATEWAY_IP = "1.1.1.1"
 HOST_IP = "1.1.1.1"
-ESP_WIFI_CSI_ENABLE = "true"
+ESP_WIFI__CSI_ENABLE = "true"
```

Comment:
looking at this changed line I wonder if a less subtle prefix would be more user-friendly. 

I can imagine users missing the second underscore and having a hard time to figure out why setting cfg-value doesn't have any effect - they won't get any indication of an error

Reply:
Sure but something like `_CFG_` is less likely to be unspotted

Reply:
Yeah I agree with @bjoernQ, `__` is likely to be missed. I think either `_CFG_` or personally I'd prefer `_CONFIG_` but either is fine with me.

Reply:
There's also the tedious option of renaming the `esp-hal-embassy` crate to `esp-embassy`, which would be consistent with literally every other crate (except the procmacros one).

(Unless there's a naming convention I'm missing)

Reply:
> (Unless there's a naming convention I'm missing)

https://github.com/[esp-rs/esp-hal-community](https://github.com/esp-rs/esp-hal-community?tab=readme-ov-file#contributing-a-crate)?tab=readme-ov-file#contributing-a-crate